docs/specs/app-specs.md

```diff
@@ -1,121 +1,186 @@
-# Dusty App Specs
-These specs are used to specify apps for you local environment. They should go in the apps folder in your specs repo.
+# App Specs
 
-## Keys
+Apps define running containers based on source code under your control. App specs tell
+Dusty how to run this container, including any dependencies the app may have on other
+apps, services, or libs. You can also specify common scripts and testing commands for
+an app in its spec.
+
+By default, Dusty manages the source for an app for you and keeps it up to date automatically.
+You can tell Dusty to use your own locally checked out copy of the source using `dusty repos`.
```

Comment:
When I first read this i thought you were taking about dusty managing dusty-specs, which I think is not worth mentioning in apps. Reading it now it seems more like you are talking about repos stuff, which is worth mentioning